packages/node-api/src/e2e.ts

```diff
@@ -35,35 +72,76 @@ describe('node-api', () => {
         });
     });
 
+    describe('assignment', () => {
+        it('should get assignmentCount', async () => {
+            const result = await getAssignmentCount(api, testCid, testCIndex);
+            expect(result.toJSON()).toStrictEqual({
+                "bootstrappers": 3,
+                "endorsees": 0,
+                "newbies": 0,
+                "reputables": 0,
+            });
+        });
+
+        it('should get assignment', async () => {
+            const assignment = await getAssignment(api, testCid, testCIndex);
+
+            // hard to test as it is randomized.
+            expect(assignment.bootstrappersReputables.m.toNumber()).toBe(3);
+            expect(assignment.bootstrappersReputables.s1.toNumber()).toBeLessThan(3);
+            expect(assignment.bootstrappersReputables.s2.toNumber()).toBeLessThan(3);
+
+            // no endorsees and newbies assigned
+            expect(assignment.endorsees.toJSON()).toStrictEqual({"m": 2, "s1": 1, "s2": 1})
+            expect(assignment.newbies.toJSON()).toStrictEqual({"m": 2, "s1": 1, "s2": 1})
+        });
+
+        it('should get meetupCount', async () => {
+            const result = await getMeetupCount(api, testCid, testCIndex);
+            expect(result.toNumber()).toBe(1);
+        });
+
+        it('should get meetupIndex', async () => {
+            for (const participant of [alice, bob, charlie]) {
+                const assignment = await getMeetupIndex(api, testCid, testCIndex, participant.address);
+                expect(assignment.toNumber()).toBe(1);
+            }
+        });
+
+        it('should get meetupLocation', async () => {
+            const location = await getMeetupLocation(api, testCid, testCIndex, testMeetupIndex);
+            expect(location.toJSON()).toStrictEqual(testCommunityParams.locations[0]);
+        });
+
+        it('should get meetupParticipants', async () => {
+            // Todo: this test only covers bootstrappers. How do we test reputables, endorsees and newbies?
+            // This might be too tedious, we'd need to go to the second meetup and also register more participants.
+            const participants = await getMeetupParticipants(api, testCid, testCIndex, testMeetupIndex);
+            expect(participants.sort().toJSON())
+                .toStrictEqual([alice.address, bob.address, charlie.address].sort());
+        });
+    });
+
     describe('rpc', () => {
         // These tests predominantly verify that we have correct rpc/type definitions
         describe('communities', () => {
             it('communities.GetAll should return empty vec', async () => {
-                const result = await api.rpc.communities.getAll();
-                // console.log(result);
-                expect(result.length).toBe(0);
+                // @ts-ignore
+                const cidNames = await api.rpc.communities.getAll();
```

Comment:
`@ts-ignore` is to remove warnings that the rpc does not contain a `communities` section. We could try to fix this. However, once we move to auto-generated types and rpc-interfaces, this will be resolved by itself.

All other `@ts-ignore`s cover the same issue.